buildpack/bomfile.go

```diff
@@ -65,24 +65,25 @@ func (b *BOMFile) mediaType() string {
 	}
 }
 
-func validateMediaTypes(bp GroupBuildpack, bomfiles []BOMFile, sbomMediaTypes []string) error {
-	contains := func(vs []string, t string) bool {
-		for _, v := range vs {
-			if v == t {
+func validateMediaTypes(bp GroupBuildpack, bomfiles []BOMFile, declaredTypes []string) error {
+	contains := func(declaredTypes []string, foundType string) bool {
```

Comment:
Should this instead use https://pkg.go.dev/mime#ParseMediaType?

Reply:
I guess so... for now we don't really need to inspect the media type with much granularity, but it could be useful in the future.

Reply:
I think when we need the ability to understand the media type that library looks good to me, but I don't see a need for it right now. Unless I'm missing something.

Reply:
Thinking on this more... maybe we should go ahead and use `mime` - it's in the standard library, and if we ever need to understand the types more granularly it will already be there i.e., we won't forget about this package and try to do our own homegrown thing. Added in 946c7064